src/test/java/com/hyperwallet/clientsdk/HyperwalletTest.java

```diff
@@ -6976,34 +7004,32 @@ public void testCreateBusinessStakeholderStatusTransition_transitionTokenSpecifi
     }
 
     @Test
-    public void testCreateBusinessStakeholderStatusTransition_successful() throws Exception {
-        HyperwalletStatusTransition transition = new HyperwalletStatusTransition();
-        transition.setFromStatus(HyperwalletStatusTransition.Status.ACTIVATED);
-        transition.setToStatus(HyperwalletStatusTransition.Status.DE_ACTIVATED);
-        transition.setCreatedOn(new Date());
-        transition.setTransition(HyperwalletStatusTransition.Status.DE_ACTIVATED);
-
-        HyperwalletStatusTransition transitionResponse = new HyperwalletStatusTransition();
+    public void testUploadDocumentBusinessStakeholder() throws Exception {
```

Comment:
Please consider renaming this test method. It seems that you are expecting an error as result.